### PR TITLE
[Pylint] fix pylint issues from test_random to test_tedd

### PR DESCRIPTION
**Pull request by tlopex**
According to #11414 , I fixed pylint issues for test_random, test_rocblas, test_rpc_proxy, test_rpc_server_device, test_rpc_tracker, test_sort, test_sparse, test_tedd in this PR.

### CODE REVIEW

File: tests/python/contrib/test_tedd.py [hunk 1]
```diff
@@ -97,39 +103,41 @@ def verify():
...
     block_x = te.thread_axis("blockIdx.x")
     thread_x = te.thread_axis("threadIdx.x")
     n = te.var("n")
     m = te.var("m")
-    l = te.var("l")
-    A = te.placeholder((n, m, l), name="A")
-    B = te.compute((n, m, l), lambda bi, bj, bk: A[bi, bj, bk] + 1, name="B")
-    r = te.reduce_axis((0, m), "r")
-    C = te.compute(
+    op_l = te.var("op_l")
```
Comment (Hzfengsy) [inline tests/python/contrib/test_tedd.py:111]: IMO, it's good to keep those concise names. I guess these names can pass the pylint through the `pylintrc`.

File: tests/python/contrib/test_tedd.py [hunk 2]
```diff
@@ -64,19 +68,21 @@ def verify():
...
 
 def test_itervar_relationship_graph():
+    """Tests itervars relationship graph"""
     n = te.var("n")
     m = te.var("m")
-    A = te.placeholder((n, m), name="A")
+    input_a = te.placeholder((n, m), name="input_a")
     k = te.reduce_axis((0, m), "k")
-    B = te.compute((n,), lambda i: te.sum(A[i, k], axis=k), name="B")
+    input_b = te.compute((n,), lambda i: te.sum(input_a[i, k], axis=k), name="input_b")
```
Comment (Hzfengsy) [inline tests/python/contrib/test_tedd.py:76]: ditto

File: tests/python/contrib/test_tedd.py (hunk 1 above)
Comment (tlopex) [reply tests/python/contrib/test_tedd.py:111]: Thanks. I will change them later.

Comment (tlopex) [reply tests/python/contrib/test_tedd.py:111]: @Hzfengsy Well, those concise names would trigger C0103(invalid-name). Like the var "l", the info is `Variable name "l" doesn't conform to '[a-z_][a-z0-9_]{2,30}$' pattern`. Is it better to use `pylint: disable=invalid-name` ?

Comment (Hzfengsy) [reply tests/python/contrib/test_tedd.py]: Sorry for the late reply, please feel free to disable invalid-name for the whole file

Comment (tlopex) [reply tests/python/contrib/test_tedd.py]: Okay, got it,

**Review by Hzfengsy — APPROVED** [review]

**Pull request merged**